src/components/FlightInfoCard/FlightInfoCard.js

```diff
@@ -0,0 +1,125 @@
+import styled from 'styled-components';
+import { CgArrowLongRight } from 'react-icons/cg';
+
+const FlightInfoCard = ({ flightInfo }) => {
+  const {
+    filghtLogo,
+    flightCompany,
+    flightNumber,
+    departureTime,
+    landingTime,
+    departureCity,
+    flightTime,
+    landingCity,
+    isDiscount,
+    personnel,
+    price,
+    baggage,
+  } = flightInfo;
+  return (
+    <FlightInfoCardWrapper>
+      <FlightLogo justify="flex-start">
+        <FlightLogoImg src={filghtLogo} alt={filghtLogo} />
+        <FlexCol>
+          <h2>{flightCompany}</h2>
+          <h3>{flightNumber}</h3>
+        </FlexCol>
+      </FlightLogo>
+
+      <FlightInfo>
+        <Flex>
+          <span>{departureTime}</span>
+          <CgArrowLongRight size={30} color="gray_300" />
+          <span>{landingTime}</span>
+        </Flex>
+        <Flex justify="center">
+          <span>{departureCity}</span>
+          <span>{flightTime}</span>
+          {landingCity}
+        </Flex>
+      </FlightInfo>
+
+      <FlightSeatsInfo>{isDiscount ? '할인석' : '일반석'}</FlightSeatsInfo>
+      <FlightSeatsInfo>{personnel}석</FlightSeatsInfo>
+
+      <FlightPriceInfo>
+        {(personnel * price).toLocaleString()}원
+      </FlightPriceInfo>
+
+      {baggage ? `무료 수하물 ${baggage}kg` : <Button>선택</Button>}
+    </FlightInfoCardWrapper>
+  );
+};
+
+export default FlightInfoCard;
+
+const FlightInfoCardWrapper = styled.section`
+  display: grid;
+  grid-template-columns: max-content 2fr 0.5fr 0.5fr max-content max-content;
+  place-items: center;
+  gap: 1em;
+  width: 40rem;
+  padding: 1em 1.25em;
+  background-color: ${props => props.theme.color.white};
+  font-size: 0.875rem;
+
+  & + & {
+    margin-top: 1em;
+  }
+`;
+
+const FlightLogo = styled.section`
+  ${props => props.theme.flex};
+  gap: 1em;
+
+  h2,
+  h3 {
+    font-size: 0.875em;
+    font-weight: 400;
+  }
+`;
+
+const FlightLogoImg = styled.img`
+  max-width: 4rem;
+`;
+
+const FlightInfo = styled.section`
+  display: grid;
+  place-items: center;
+
+  > *:nth-child(2) {
+    font-size: 0.8em;
+    color: ${props => props.theme.color.gray_500};
+  }
+`;
+
+const FlightSeatsInfo = styled.span`
+  font-size: 0.875em;
+`;
+
+const Button = styled.button`
+  padding: 0.5em 1em;
+  background-color: ${props => props.theme.color.primary_400};
+  color: ${props => props.theme.color.white};
+  font-size: 1rem;
+  cursor: pointer;
+
+  &:hover {
+    background-color: ${props => props.theme.color.primary_500};
+  }
+`;
+
+const FlightPriceInfo = styled.h2`
+  font-size: 1.5em;
+`;
+
+const Flex = styled.div`
+  ${props => props.theme.flex};
```

Comment:
```suggestion
  ${({ theme }) => theme.flex};
```
작은 부분이긴 한데 얘도 구조분해 할당 가능합니다

Reply:
({theme}) 으로 구조분해 할당하면 불필요한 props 줄이는 장점이 있지만 저는 ${({theme}) => ... } 앞에 괄호가 너무 많이 들어가 가독성이 그렇게 좋다고 생각하지 않아 props를 사용했습니다. 
프로젝트 진행하다 보니 불필요한 props가 너무 많아져서 지우는 게 맞는 것 같습니다. 리뷰 주신대로 수정해 보겠습니다! 감사합니다!